src/genie/libs/parser/iosxe/show_sdwan_control_connections.py

```diff
@@ -0,0 +1,91 @@
+# Metaparser
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Or, Optional
+from genie import parsergen
+from pprint import pprint
+import re
+
+# ===========================================
+# Schema for 'show sdwan control connections'
+# ===========================================
+
+
+class ShowSdwanControlConnectionsSchema(MetaParser):
+
+    """ Schema for "show sdwan control connections" """
+
+    schema = {
+        "control_connections": {
+            Any(): {
+                Any(): {
+                    "controller_group_id": str,
+                    "domain_id": str,
+                    "peer_private_ip": str,
+                    "peer_private_port": str,
+                    "peer_protocol": str,
+                    "peer_public_ip": str,
+                    "peer_public_port": str,
+                    "peer_type": str,
+                    "proxy_state": str,
+                    "site_id": str,
+                    "uptime": str,
+                },
+            },
+        },
+    }
+
+
+# ===========================================
+# Parser for 'show sdwan control connections'
+# ===========================================
+
+
+class ShowSdwanControlConnections(ShowSdwanControlConnectionsSchema):
+
+    """ Parser for "show sdwan control connections" """
+
+    cli_command = "show sdwan control connections"
+
+    def cli(self, output=None):
+        if output is None:
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output
+
+        parsed_dict = {}
+
+        p1 = re.compile(
```

Comment:
Added sample output from my device. it seems 'state' = 'up' is missing in regex.
```suggestion
        # vsmart  dtls 4.4.4.70        100        1      10.10.20.70                             12446 10.10.20.70                             12446 default         No    up     2:13:54:01  0
        p1 = re.compile(
```

Reply:
any chance to update this?

Reply:
done